src/db/entity/Member.ts

```diff
@@ -0,0 +1,31 @@
+import { Entity, ObjectIdColumn, ObjectID, Column } from "typeorm";
+
+@Entity()
+export class Member {
+  @ObjectIdColumn()
+  _id: ObjectID;
+
+  @Column()
+  user_id: string;
+
+  @Column()
+  nickname: string;
+
+  @Column()
+  image_url: string;
```

Comment:
For some reason in higher-level languages like TS, Java, C#, I really hate snake case, so if you are the same way you can change the Column decorator to `@Column({ name: "user_id" }) userId: string;` Up to you if you wanna keep snake case